### DISCUSSION TITLE
No Free Space

### DISCUSSION THREAD

**Discussion by loganvandyke** [grahampugh/erase-install#185] [Q&A]
Hello,

I keep getting this error on about half the Macs I try to upgrade using Jamf from Big Sur (11.6.0, 11.6.1, 11.6.2) to Monterey 12.1. I am using the installed package and calling this command: /Library/Management/erase-install/erase-install.sh --force-curl --update --reinstall --os=12 --depnotify

The Macs do have plenty of free space on them.

Results:

Print: Entry, "AppleLanguages:0", Does Not Exist

   [erase-install] v25.0 script execution started: Sat Jan  8 00:01:13 EST 2022
   [erase-install] Caffeinating this script (pid=63936)
   [check_free_space] ERROR - 0 GB free/purgeable disk space detected
0   [erase-install] 'caffeinate' ended
/Library/Management/erase-install/erase-install.sh: line 945: 63947 Terminated: 15          /usr/bin/caffeinate -dimsu -w $pid
   [erase-install] 'jamfHelper' ended

**Comment by Trikster-Online** [comment]
Anyone have an update on this? I am getting a similar error when trying to upgrade from Catalina to Monterey. I have 350GB available free space on the drive...

**Reply by jfiliceatcsumb** [reply]
I am getting this as well trying to erase-install from 10.15.7 to 12.5. Disk has over 150 GB free.

**Reply by Trikster-Online** [reply]
I seem to have found a work around to this. I had to reboot the target computers a couple times for this to work. Once it got going, it worked great. Seems that sometimes the download of the macOS installer doesn't unmount itself and when we run the script to install the new version of macOS, it tries to install to that mounted installer. A reboot or two did the trick.

**Reply by jfiliceatcsumb** [reply]
I realized I was trying to do this on Macs with no users signed in. After signing into a local account, it worked as expected. <facepalm>

**Reply by Trikster-Online** [reply]
Yeah, I ran into that as well. Wish there was a way for this to work without a user signed in...Especially on the M1/T2 Macs. Dialogs to authenticate a user when you have a couple hundred remote Mac's is not easy.
